build_rules.gradle

```diff
@@ -389,6 +389,14 @@ ext.applyGoNature = {
     goVersion = '1.9'
   }
 
+  repositories {
+    golang {
+      // Gogradle doesn't like thrift: https://github.com/gogradle/gogradle/issues/183
+      root 'git.apache.org/thrift.git'
+      emptyDir()
+    }
+  }
+
   // GoGradle fails in a parallel build during dependency resolution/installation.
   // Force a dependency between all GoGradle projects during dependency resolution/installation.
   // TODO: Figure out how to do this by automatically figuring out the task dependency DAG
```

Comment:
gogradle 0.8.1 allegedly solves this problem

If this isn't needed anymore, can you remove it in this PR or a subsequent one?

Reply:
Sure. Let me try it in a subsequent PR.